src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/controller/OrchestrationSessionsController.kt

```diff
@@ -147,6 +147,12 @@ class OrchestrationSessionsController(private val visitSchedulerSessionsService:
       description = "Advances the available visits slots sought from date by n days. Defaults to 0 if not passed.",
     )
     advanceFromDateByDays: Int? = 0,
```

Comment:
should we remove this parameter if we are adding a fromDateOverride?

Reply:
As mentioned in slack after discussion. this field is used specifically by PVB, it adds n days to the current opening booking window. As PVB wants it to be the prison config date + 1 day, they will simply not pass in fromDateOffset, and rely on the prison's configurations (pre-existing functionality).